water.mss

```diff
@@ -244,6 +244,23 @@
     }
   }
 
+  [attraction = 'water_slide'] {
+    [zoom >= 15] {
+      [zoom >= 17] {
+        bridgecasing/line-color: saturate(darken(@water-color, 40%), 30%);
```

Comment:
Actually, I'm not sure what the prefix "bridgecasing" references to here.  Is it really needed?